src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  font-size: 11px;
+  font-weight: 400;
+  font-family: $roboto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: $card-width;
+  padding: 20px 14px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    width: $picture-width;
+    height: 134px;
+    margin: 9px auto 45px;
+  }
+  
+  &__text {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 15px;
+    padding-bottom: 6px;
+  }
+  
+  &__code {
+    padding-bottom: 15px;
+  }
+  
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 20px;
+  }
+  
+  &__list-stars {
+    display: flex;
+  }
+  
+  &__item {
+    width: $star-size;
+    height: $star-size;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  
+  &__item:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    padding-bottom: 20px;
+  }
+  
+  &__price-number {
+    font-size: 16px;
+    font-weight: 700;
+  }
+  
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    min-height: 40px;
+    border: 1px solid $blue-color;
+    color: $white-color;
+    background-color: $blue-color;
+    border-radius: 5px;
+  }
+  
+  &__button:hover {
+    color: $blue-color;
+    background-color: $white-color;
+  }
+
+  &--space-between {
+    margin: 0 $card-margin 50px;
+  }
```

Comment:
add justify-content: space-between to the card, no need to hardcode margins for each card

Reply:
for vertical space between card rows add mix class for each card with margin-bottom
block and modifier can't have outer spacing styles
https://www.youtube.com/watch?v=JOJ-mENrg_s

Reply:
--space-between is obsolete, remove it